src/dotnet/APIView/APIView/MethodAPIV.cs

```diff
@@ -40,12 +41,14 @@ public MethodAPIV(IMethodSymbol symbol)
             {
                 this.Name = symbol.ContainingType.Name;
                 this.IsConstructor = true;
-                this.ReturnType = "";
+                this.ReturnType = new TypeReference();
```

Comment:
why not null?

Reply:
I didn't want to accidentally trigger any null pointer exceptions - I'm not sure what the standard practice for handling null values is in C#